src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py

```diff
@@ -163,17 +170,26 @@ def make_or_load_nwbfile(
         "'nwbfile_path' exists at location, 'overwrite' is False (append mode), but an in-memory 'nwbfile' object was "
         "passed! Cannot reconcile which nwbfile object to write."
     )
-
+    if overwrite is False and backend == "zarr":
+        # TODO: remove when https://github.com/hdmf-dev/hdmf-zarr/issues/182 is resolved
+        raise NotImplementedError("Appending a Zarr file is not yet supported!")
     load_kwargs = dict()
     success = True
-    file_initially_exists = nwbfile_path_in.is_file() if nwbfile_path_in is not None else None
+    file_initially_exists = nwbfile_path_in.exists() if nwbfile_path_in is not None else None
     if nwbfile_path_in:
-        load_kwargs.update(path=nwbfile_path_in)
+        load_kwargs.update(path=str(nwbfile_path_in))
```

Comment:
I should push that PR to zarr so they take paths.

Reply:
Requested in https://github.com/hdmf-dev/hdmf-zarr/issues/181, really hoping neither of us would need to spend time on it and just let the LBNL team handle it, but who knows 🤷